requirements.txt

```diff
@@ -3,7 +3,8 @@ eventlet
 Jinja2>=2.11 # BSD License (3 clause)
 jsonschema!=2.5.0,<3.0.0,>=2.0.0 # MIT
 # networkx v2.6 does not support Python3.6.  Update networkx to match st2
-networkx>=2.5.1,<2.6
+networkx>=2.5.1,<2.6; python_version < '3.7'
+networkx>=2.6,<3; python_version >= '3.7'
```

Comment:
I updated it to a more strict `>=2.6` as https://security.snyk.io/vuln/SNYK-PYTHON-NETWORKX-1062709 suggests